examples/hello_rust/hello.typ

```diff
@@ -9,15 +9,3 @@
   // p.will_panic()  // Fails compilation
   // p.returns_err() // Fails compilation with an error message
 }
-
```

Comment:
Why does this need to be removed?

Reply:
Oh right – this was originally because I was going to avoid having the WASI be generated always in CI, since that would avoid the need for installing Rust on all the jobs. But later I just gave in.

However, I do think there is some value in allowing people to try out the examples without building both WASI and non-WASI versions? I feel like in practice, people will either know they want WASI, or know they don’t care about it, and so it’s unlike that they’ll want to try out both methods before `hello.typ` can build.

But I can always change it back.

Reply:
I think this is reasonable, since we still test with both wasi and non-wasi version.